### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by beatngu13**

### CODE REVIEW

File: src/test/java/de/retest/recheck/testng/ExecuteTest.java [hunk 1]
```diff
@@ -13,27 +13,23 @@
 
 	private static class DummyClass {
 
-		private RecheckLifecycle someField;
-		private RecheckLifecycle otherField;
+		@SuppressWarnings( "unused" )
+		public final Object someObject = new Object();
+		protected final RecheckLifecycle someLifecycle = mock( RecheckLifecycle.class );
```
Comment (briemla) [inline src/test/java/de/retest/recheck/testng/ExecuteTest.java]: Why is this field `protected`?

Comment (beatngu13) [reply src/test/java/de/retest/recheck/testng/ExecuteTest.java]: I thought I play around with different field visibilities for testing. Do you find it confusing?

Comment (briemla) [reply src/test/java/de/retest/recheck/testng/ExecuteTest.java]: I was a bit confused if it was an accident or not. The intention to have different visibility levels for testing could be reflected in the variable names: `publicObject`, `protectedLifecycle` and `privateLifecycle`.

Comment (beatngu13) [reply src/test/java/de/retest/recheck/testng/ExecuteTest.java]: Good idea, let me change that!

**Pull request merged**